subsys/shell/Kconfig.backends

```diff
@@ -30,8 +30,19 @@ config SHELL_PROMPT_UART
 # Workaround for not being able to have commas in macro arguments
 DT_CHOSEN_Z_SHELL_UART := zephyr,shell-uart
 
+config UART_SHELL_USE_CDC_ACM
+	bool "Deliver Shell through USB CDC AMC"
+	default n
```

Comment:
This line is obsolete, `n` is a default choice.

Reply:
I think changes in Kconfig.backends are not necessary at all. Adding CONFIG_SHELL_ON_DEV_NAME="CDC_ACM_0" to the configuration overlay/file of an application/sample should also work.

Reply:
The only reason I introduced the new key in Kconfig was because a change to `SYS_INIT` in `shell_uart.c` has to be done at compile time (and then the same key can also control whether `usb_enable` is triggered or not. Do you have any idea on how this can be accomplished differently?